### PR TITLE
Consolidate some nodes from Logic and Formula to Math 

### PR DESCRIPTION
**Pull request by fanwgwg**
### Purpose
*[DYN-720](https://jira.autodesk.com/browse/DYN-720) Consolidate below mentioned nodes from Logic and Formula to Math category.*

**Old Nodes:**
Logic > And
Logic > Or
Logic > XOr
Math.DivRem
Formula > Evaluate
Math.DegreesToRadians
Math.RadiansToDegrees

**New Nodes:**
Math.And
Math.Or
Math.XOr
Math.DivRem
Math.EvaluateFormula
Math.DegreesToRadians
Math.RadiansToDegrees

### Declarations

Check these if you believe they are true

- [ ] The code base is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions), and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@riteshchandawar

### CODE REVIEW

File: src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml [hunk 1]
```diff
@@ -38,4 +38,20 @@
...
+    <oldName>DSCore.Logic.Xor</oldName>
+    <newName>DSCore.Math.Xor</newName>
+  </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.Logic.And</oldName>
+    <newName>DSCore.Math.And</newName>
+  </priorNameHint>
+  <priorNameHint>
+    <oldName>DSCore.Logic.Or</oldName>
+    <newName>DSCore.Math.Or</newName>
```
Comment (gregmarr) [inline src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml]: Unless I'm missing something, the And and Or functions don't exist, just Xor.

Comment (gregmarr) [inline src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml]: They do exist in `CoreNodeModels.Logic.`
https://github.com/DynamoDS/Dynamo/blob/master/test/core/customast/and-test.dyn#L4

Comment (gregmarr) [inline src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml]: Ah, there it is:
https://github.com/DynamoDS/Dynamo/blob/master/src/Libraries/CoreNodeModels/Logic/AndOr.cs#L80

**Comment by fanwgwg** [comment]
Close this because a wrong branch has been referenced. New PR at #7757

**Pull request closed without merging**